cmd/cloud-controller-manager/app/options/BUILD

```diff
@@ -17,7 +17,6 @@ go_library(
         "//cmd/cloud-controller-manager/app/config:go_default_library",
         "//cmd/controller-manager/app/options:go_default_library",
         "//pkg/controller:go_default_library",
-        "//pkg/features:go_default_library",
```

Comment:
I believe we want to change this to 
"//staging/src/k8s.io/apiserver/pkg/features:go_default_library",